server/src/workspace/workspace-query-builder/factories/args-alias.factory.ts

```diff
@@ -50,7 +52,12 @@ export class ArgsAliasFactory {
           const mappedKey = fieldMetadata.targetColumnMap[subKey];
 
           if (mappedKey) {
-            newArgs[mappedKey] = subValue;
+            // Currency metadata type value should be a string to match graphql bigInt requirements
```

Comment:
@martmull Instead of doing that you should use the `BigIntScalarType` that is already in the project:

```typescript
export const BigIntScalarType = new GraphQLScalarType({
  name: 'BigInt',
  description:
    'The `BigInt` scalar type represents non-fractional signed whole numeric values.',
  serialize(value: bigint): string {
    return value.toString();
  },
  parseValue(value: string): bigint {
    return BigInt(value);
  },
  parseLiteral(ast): bigint | null {
    if (ast.kind === 'IntValue') {
      return BigInt(ast.value);
    }

    return null;
  },
});
```